src/org/labkey/test/pages/list/EditListDefinitionPage.java

```diff
@@ -69,6 +62,54 @@ public static EditListDefinitionPage beginAt(WebDriverWrapper driver, String con
         return new EditListDefinitionPage(driver.getDriver());
     }
 
+    public void waitForPage()
+    {
+        waitFor(()-> elementCache().listPropertiesHeaderLoc.existsIn(getDriver()),
+            "The page did not render in time", WAIT_FOR_PAGE);
+    }
+
+    public EditListDefinitionPage setListName(String listName)
+    {
+        expandPropertiesPane();
+        elementCache().nameInput().set(listName);
+        return this;
+    }
+
+    public EditListDefinitionPage setDescription(String description)
+    {
+        expandPropertiesPane();
+        elementCache().descriptionInput().set(description);
+        return this;
+    }
+
+    public AdvancedListSettingsDialog getAdvancedListSettings()
```

Comment:
I generally prefer to name methods `getSomething` only if they don't change the page state. They are also, usually, terminal methods (e.g. `getDescription`).
```suggestion
    public AdvancedListSettingsDialog openAdvancedListSettings()
```